setup.go

```diff
@@ -31,6 +48,23 @@ const (
 	envLoggingFile = "GOLOG_FILE" // /path/to/file
 )
 
+type Config struct {
+	// Format overrides the default format of the log output which is line based text with color codes
+	// nocolor: line based text format without color codes
+	// json: outputs logs in json
+	Format string
```

Comment:
nit: I'd consider using an enum:

```go
type LogFormat int
const (
  ColorizedOutput LogFormat = iota
  PlaintextOutput
  JSONOutput
)
```

(or something like that)

Reply:
Done